README.md

```diff
@@ -1,20 +1,21 @@
 video-mode-plugin
 =================
 
-A BrightAuthor plugin that changes the video resolution, frame rate, color space and color depth.
+A BrightSign plugin that changes the video resolution, frame rate, color space and color depth.
 
-# Usage
-
-1. Add videomode_plugin.brs to BrightAuthor (File > Presentation Properties > Autorun)
-1. Add a plugin message wherever you want to send it: Advanced tab > Add Command > Send > Send Plugin Message, pick the plugin you added in step 1, and add the text of the the plugin message e.g. "brightsign!videomode!1920x1080x60i!444!10bit" (format below)
+SetSingleScreenVideoMode.brs for non-series 5 single screen players.
+SetMultiScreensVideoMode.brs for series 5 multi screens players.
 
-The format of command is a set of values separated by exclamation characters:
+# Usage
 
-    brightsign!videomode!<resolution/frame rate>!<color space>!<color depth>
```

Comment:
Should we still be supporting this approach? There may be beneficial use cases customers to support a "plugin message" approach for how to "set a video mode".

Reply:
Added the approach of Plugin Message for videomode back.